autofs-5.0.7 - document browse option in man page

From: Ian Kent <raven@themaw.net>

The "browse" option has remained undocumented for a long time.
Finally add a section for it to auto.master(5) making special
note of the potential performance implications.
---
 CHANGELOG            |    1 +
 man/auto.master.5.in |   11 +++++++++++
 2 files changed, 12 insertions(+)

--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -196,6 +196,7 @@
 - fix incorrect append options description in README.v5-release.
 - fix mistake in assignment.
 - use open(2) instead of access(2) to trigger dependent mounts.
+- document browse option in man page.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/man/auto.master.5.in
+++ autofs-5.0.5/man/auto.master.5.in
@@ -145,6 +145,17 @@ multiple file systems should be mounted
 is given, no file system is mounted at all if at least one file system
 can't be mounted.
 .TP
+.I "[no]browse"
+This is an autofs specific option that is a pseudo mount option and
+so is given without a leading dash. Use of the browse option pre-creates
+mount point directories for indirect mount maps so the map keys can be
+seen in a directory listing without being mounted. Use of this option
+can cause performance problem if the indirect map is large so it should
+be used with caution. The internal program default is to enable browse
+mode for indirect mounts but the default installed configuration overrides
+this by setting BROWSE_MODE to "no" because of the potential performance
+problem.
+.TP
 .I "nobind"
 This is an autofs specific option that is a pseudo mount option and
 so is given without a leading dash. It may be used either in the master
